aws/resource_aws_dlm_lifecycle_policy.go

```diff
@@ -50,6 +50,12 @@ func resourceAwsDlmLifecyclePolicy() *schema.Resource {
 							Required: true,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
+									"copy_tags": {
```

Comment:
Can you please add this argument's documentation to `website/docs/r/dlm_lifecycle_policy.html.markdown`? Thanks!